Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -77,12 +77,28 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
-    new_version = args.version
-    new_version = :latest if ["latest", ":latest"].include? new_version
-    new_version = Cask::DSL::Version.new(new_version) if new_version.present?
-    new_base_url = args.url
-    new_hash = args.sha256
-    new_hash = :no_check if ["no_check", ":no_check"].include? new_hash
+    new_version = unless (new_version = args.version).nil?
+      raise UsageError, "`--version` must not be empty." if new_version.blank?
```

Comment:
Are there any cases where an empty argument is allowed? If not, it probably makes sense to add this error to `CLI::Parser` instead.

Reply:
Opened an issue for this here: https://github.com/Homebrew/brew/issues/15115